### PR TITLE
Youtube video player

### PR DESCRIPTION
**Pull request by shadrachodek**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  box-shadow: 0 8px 15px 0 rgba(0,0,0,0.1);
+  transition: 0.3s;
+}
+
+
+main{
+  /* float: left; */
+  display: inline-block;
+  width: 70%;
+  /* background-color: aquamarine; */
```
Comment (samgaco) [inline styles.css]: It is better to delete commented code :smiley:

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+.upload {
+  width: 1.3em;
+  margin-right: 15px;
+  position: relative;
+  left: 90px;
+}
+
+.youtube-logo {
+  width: 90px;
+  /* margin-right: 1em; */
```
Comment (samgaco) [inline styles.css]: Commented code should be gone :1234:

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @shadymedy and @trekab :vulcan_salute: 

### Your project is looking neat! well done on that :clap: Anyways, there are some things we still can work on:

* The **search button** at the left of the search bar should also be enclosed within the `<button>` tag.
* Try to **improve the readme file**, add your contacts, a better description of the project and a live demo of the site https://raw.githack.com/... you can also find inspiration in template examples like [this one.](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
* **Correct the errors** given to you by uploading the index file to the [validator](https://validator.w3.org)
* Each section (video, comments, sidebar, navigation bar) has it's own container (`<div>` or similar) for the embedded video (point 5 in projects description).

### Please :pray: address these issues and submit back for another code review when you are done. Keep up the great work :muscle: !!

**Review by Oghenebrume50 — APPROVED** [review]
Hi team good job, it is a wrap for this one :smiley: please merge to master then submit the project completion form, :+1:

File: styles.css (hunk 1 above)
Comment (trekab) [reply styles.css]: Done!

**Pull request merged**